### PR TITLE
Add dashboard chart

### PR DESCRIPTION
**Pull request by kulczy**
Zrzut ekranu 2020-01-21 o 14 55 00

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Dashboard/_chart.html.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% set labels, values = [], [] %}
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Dashboard/_chart.html.twig]: Can you provide dummy data?

File: src/Sylius/Bundle/AdminBundle/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -44,5 +44,6 @@ sylius:
                     header: Sales
                     orders: Orders
     ui:
+        annual_summary: Annual summary
```
Comment (CoderMaggie) [inline src/Sylius/Bundle/AdminBundle/Resources/translations/messages.en.yml]:
```suggestion
        sales_summary: Sales summary
```

File: UPGRADE-1.7.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ Update your `package.json` in order to add `slick-carousel` :
 {
   "dependencies": {
     "babel-polyfill": "^6.26.0",
++   "chart.js": "^2.9.3",
```
Comment (pamil) [inline UPGRADE-1.7.md]: Line 9 of `UPGRADE.md` should be updated to reflect this change.

**Comment by pamil** [comment]
Thanks, Szymon! :1st_place_medal:

**Pull request merged**